crypto/SMT/utils.ts

```diff
@@ -19,6 +19,7 @@ export const remove0x = (str: string): string => {
  * @returns the string with "0x".
  */
 export const add0x = (str: string): string => {
+    str = str.padStart(64,"0")
```

Comment:
It is used in the database version. If the string is processed as a `string` instead of a `BigInt`.
I didn't notice there is a `toCompleteHexString`. I will use this function in the database version. Thanks.